metrics/storage/fio_job.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#  Description of the test:
+#  This test measures the bandwidth consumed by a specif I/O
+#  operation (read, write, random-read etc) under certain configuration.
+#  The result is obtained using FIO tool, it runs inside of a container.
+
+set -e
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+# Fio job default configuration
+BLOCK_SIZE="4k"
+DEBUG_FILE="debug.log"
+ENGINE="sync"
+FILE_NAME="/home/fio_test"
```

Comment:
I'll let @MarioCarrilloA answer as well, but iirc, where you place the test file directly influences which type of container mount point you are actually testing the bandwidth of (9p, union, ram etc.).
I believe we are using the `/home/` here to force use of a specific mount point, and will let @MarioCarrilloA explain (and then probably add a comment to the file to make that clearer)